### PR TITLE
Fix/topic and target pk

### PR DESCRIPTION
**Pull request by JorgeMichelena**

### CODE REVIEW

File: targets/validators.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    longitude = value.y
+    if latitude>180 or latitude<-180:
+        raise ValidationError(
+            _('Latitude must be a real number between -180 and 180'), 
+        )
+    if longitude>90 or longitude<-90:
+        raise ValidationError(
+            _('Longitude must be a real number between -90 and 90'), 
+        )
+
```
Comment (kaozdl) [inline targets/validators.py]: Missing newline here

File: users/migrations/0002_user_gender.py [hunk 2]
```diff
@@ -1,18 +0,0 @@
-# Generated by Django 3.0.3 on 2020-02-19 17:20
```
Comment (kaozdl) [inline users/migrations/0002_user_gender.py]: Why did you delete this migration?

File: targets/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
-# Generated by Django 3.0.3 on 2020-02-28 17:55
```
Comment (kaozdl) [inline targets/migrations/0001_initial.py]: Why is the migration altered to create a new model?

File: users/migrations/0002_user_gender.py (hunk 2 above)
Comment (JorgeMichelena) [reply users/migrations/0002_user_gender.py]: Had an issue when I changed the name field in the topic model from being the pk to being just a unique field.
Django asked me to give a default value to the new pk column and when I tried to set a default value for it a message error was shown saying it now had to many default values.
Didn't know how to handle the situation so I just deleted all the migrations and created them all over again.

File: targets/migrations/0001_initial.py (hunk 3 above)
Comment (JorgeMichelena) [reply targets/migrations/0001_initial.py]: See answer abover.
